src/App.tsx

```diff
@@ -57,7 +57,15 @@ export const App = () => {
   const { width, ref } = useResizeDetector<HTMLDivElement>();
   const isSmallScreen = width !== undefined && width < 768;
 
-  const { data: nextcladeDatasetInfo } = useQuery(() => fetchNextcladeDatasetInfo(), []);
+  const queryFunction =
+    sequenceDataSource === 'gisaid'
+      ? fetchNextcladeDatasetInfo
+      : () => {
+          return Promise.resolve({ name: 'notNextcladeDatasetInfo', tag: null } as NextcladeDatasetInfo);
+        };
```

Comment:
Nitpicking: makes it nicer to read
```suggestion
      : () => Promise.resolve({ name: 'notNextcladeDatasetInfo', tag: null } as NextcladeDatasetInfo);
```